src/objectTranslations/Address__c-de.objectTranslation

```diff
@@ -1,522 +1,518 @@
-      <translation>24</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>25</masterLabel>
-      <translation>25</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>26</masterLabel>
-      <translation>26</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>27</masterLabel>
-      <translation>27</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>28</masterLabel>
-      <translation>28</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>29</masterLabel>
-      <translation>29</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>30</masterLabel>
-      <translation>30</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>31</masterLabel>
-      <translation>31</translation>
-    </picklistValues>
-  </fields>
-  <fields>
-    <name>Seasonal_Start_Month__c</name>
-    <help>Der Monat im Jahr, ab dem diese Adresse die Standardadresse ersetzt.</help>
-    <label>Monat des Saisonstarts</label>
-    <picklistValues>
-      <masterLabel>1</masterLabel>
-      <translation>1</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>2</masterLabel>
-      <translation>2</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>3</masterLabel>
-      <translation>3</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>4</masterLabel>
-      <translation>4</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>5</masterLabel>
-      <translation>5</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>6</masterLabel>
-      <translation>6</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>7</masterLabel>
-      <translation>7</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>8</masterLabel>
-      <translation>8</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>9</masterLabel>
-      <translation>9</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>10</masterLabel>
-      <translation>10</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>11</masterLabel>
-      <translation>11</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>12</masterLabel>
-      <translation>12</translation>
-    </picklistValues>
-  </fields>
-  <fields>
-    <name>Seasonal_Start_Year__c</name>
-    <help>Das Jahr, ab dem diese Adresse die Standardadresse ersetzt.</help>
-    <label>Jahr des Saisonstarts</label>
-  </fields>
-  <layouts>
-    <layout>HEDA Address Layout</layout>
-    <sections>
-      <label>Benutzerdefinierte Links</label>
-      <section>Custom Links</section>
-    </sections>
-  </layouts>
-  <nameFieldLabel>Adressen-ID</nameFieldLabel>
-  <standardFields>
-    <name>Name</name>
```

Comment:
@MatthewBlanski looks like this one is missing here, I will check all the translations files again and push the changes in a single commit.

Reply:
@MatthewBlanski Only after doing a few I realized, looks like <standardFields> <name>...</name></standardField> is replaced with <nameFieldLabel>....</nameFieldLabel>, that is causing the difference in line count.